co2eq/meal.js

```diff
@@ -20,6 +20,14 @@ const CARBON_INTENSITY = { // kgCO2eq / kg
   'Milk': 1.9,
   'Fruit': 1.1,
   'Lentils': 0.9,
+  'Pasta': 1.2,
```

Comment:
Can you add a comment above this line saying that all the next elements come from a certain publication (and mention that publication?)